app/src/main/java/org/mozilla/fenix/settings/DataChoicesFragment.kt

```diff
@@ -18,30 +18,33 @@ import org.mozilla.fenix.ext.settings
 import org.mozilla.fenix.ext.showToolbar
 
 /**
- * Lets the user toggle telemetry on/off.
```

Comment:
Just to confirm, this is the only real change in this file correct?  The rest is only refactoring for readability?

Reply:
No, see below comment

Reply:
Yes, I just want to confirm for `DataChoicesFragment` these are just refactoring right? I can't see a difference and I'm not sure if I'm reading it right.

Reply:
Just refactoring

Reply:
I'll take this out